types/fingerprintjs__fingerprintjs/fingerprintjs__fingerprintjs-tests.ts

```diff
@@ -0,0 +1,135 @@
+import * as fingerprint2 from 'fingerprintjs2';
```

Comment:
How does it really passed the CI?
```suggestion
import * as fingerprint2 from '@fingerprintjs/fingerprintjs';
```

Reply:
Probably because diff between the packages at this point is minimal, npm says 2 patch versions.  Good catch.